tests_e2e/tests/lib/vm_extension.py

```diff
@@ -99,9 +99,9 @@ def enable(
                 extension_parameters
             ).result(timeout=_TIMEOUT))
 
-        if result.provisioning_state != 'Succeeded':
```

Comment:
This check is failing from time to time because we get 'Updating'; that should not be reported as a failure.